website/content/en/preview/concepts/disruption.md

```diff
@@ -112,19 +112,19 @@ For spot nodes, Karpenter only uses the deletion consolidation mechanism.  It wi
 {{% /alert %}}
 
 ### Drift
+#### Specified Drift  
```

Comment:
This reads very strangely to me. Thoughts on just `Spec Drift`, since its about changed applied to `spec`?

@jonathan-innis any thoughts?

Reply:
Agreed. I think that Spec drift makes more sense

Reply:
I originally was against using Specified/Spec, but I didn't have a better alternative.

Spec Drift sounds like Drift of the object's spec field, like NodeClassSpec.
I could imagine reading this and being confused why it's called that when SGs, Subnets, and AMIs and NodeRequirements aren't included in this.

Reply:
How would y'all feel if we dropped the naming all together? Just say drift for the general drift, special drift, and no drift

Reply:
Talked with @engedaam, we're thinking it might be better to just call out "Drift has special behaviors for certain fields, but otherwise generally works like [Specified Drift]"

Reply:
> being confused why it's called that when SGs, Subnets, and AMIs and NodeRequirements aren't included in this

I'm on-board with this except for the fact that I would still want to call it "Spec Drift". We're referencing the spec of an object and diffing the hash of that spec  to determine if something has drifted. Like you said, I think saying something like, "This is how drift works generally for Spec Drift, but there are some exceptions that are Discovered and here's the reason why these are different."